mappings/net/minecraft/village/VillagerData.mapping

```diff
@@ -1,11 +1,25 @@
 CLASS aub net/minecraft/village/VillagerData
+	FIELD a LEVEL_EXPERIENCE [I
 	FIELD b type Lauf;
 	FIELD c profession Laud;
 	FIELD d level I
+	METHOD <init> (Lauf;Laud;I)V
+		ARG 1 type
+		ARG 2 profession
+		ARG 3 level
 	METHOD a getType ()Lauf;
 	METHOD a withLevel (I)Laub;
+		ARG 1 level
 	METHOD a withProfession (Laud;)Laub;
+		ARG 1 profession
 	METHOD a withType (Lauf;)Laub;
+		ARG 1 type
 	METHOD a serialize (Lcom/mojang/datafixers/types/DynamicOps;)Ljava/lang/Object;
 	METHOD b getProfession ()Laud;
+	METHOD b getLowerLevelExperience (I)I
+		ARG 0 level
 	METHOD c getLevel ()I
+	METHOD c getUpperLevelExperience (I)I
+		ARG 0 level
+	METHOD d isLevelValid (I)Z
```

Comment:
I remember this one returns false for stone level. Is this `hasLevelledUp`?

Reply:
Oh, potentially. Will have to check that, thanks!

Reply:
Seems the minimum level value is `1`